engine/src/main/java/org/hawkular/alerts/engine/impl/ispn/IspnActionsServiceImpl.java

```diff
@@ -61,32 +95,312 @@ public void setActionsCacheManager(ActionsCacheManager actionsCacheManager) {
         this.actionsCacheManager = actionsCacheManager;
     }
 
-    public void setExecutor(ExecutorService executor) {
```

Comment:
The executor has the meaning to not block the engine thread while writting in the backend or in actions, when processing actions.
So I would leave it.
Once that we start the round of itest on Ispn backend we can decide if we refactor it but I wouldn't introduce this change now.